### PR TITLE
docs: `title` component translation

### LINKED ISSUE #690: `<title>`
Page: https://id.react.dev/reference/react-dom/components/title

### PR DESCRIPTION
**Pull request by aroyan**
Closes  #690  

## Description

Translate the `<title>` page.
Page URL: https://id.react.dev/reference/react-dom/components/title

### CODE REVIEW

File: src/content/reference/react-dom/components/title.md [hunk 1]
```diff
@@ -5,17 +5,17 @@ canary: true
 
 <Canary>
 
-React's extensions to `<title>` are currently only available in React's canary and experimental channels. In stable releases of React `<title>` works only as a [built-in browser HTML component](https://react.dev/reference/react-dom/components#all-html-components). Learn more about [React's release channels here](/community/versioning-policy#all-release-channels).
+Ekstensi React untuk `<title>` saat ini hanya tersedia di saluran canary dan eksperimental React. Pada rilis stabil React, `<title>` hanya berfungsi sebagai [komponen HTML bawaan peramban](https://react.dev/reference/react-dom/components#all-html-components). Pelajari lebih lanjut tentang [saluran rilis React di sini](/community/versioning-policy#all-release-channels).
```
Comment (resir014) [inline src/content/reference/react-dom/components/title.md:8]:
```suggestion
Ekstensi React untuk `<title>` saat ini hanya tersedia di kanal *canary* dan eksperimental React. Pada rilis stabil React, `<title>` hanya berfungsi sebagai [komponen HTML bawaan peramban](https://react.dev/reference/react-dom/components#all-html-components). Pelajari lebih lanjut tentang [saluran rilis React di sini](/community/versioning-policy#all-release-channels).
```

saluran -> kanal

**Review by resir014 — COMMENTED** [review]
One minor change, please check! @aroyan

**Comment by aroyan** [comment]
> One minor change, please check! @aroyan

updated, thanks

**Comment by resir014** [comment]
> for this link title, should this updated to "kanal" too?

@aroyan yes.

**Review by r17x — CHANGES_REQUESTED** [review]
use `*` instead of `_` for _italic_

**Review by r17x — APPROVED** [review]

**Pull request merged**